tests/srw_test.cpp

```diff
@@ -512,23 +514,6 @@ static void test_localnode(session &client, const std::vector<int> &groups, int
 		BOOST_REQUIRE_NO_THROW(result = std::move(future.get()));
 	}
 
-	// BOOST_REQUIRE_EQUAL(std::get<0>(read_result), std::get<0>(write_result));
-    //FIXME: can't compare whole read_result to a write_result
-    // because right now read_result.data_offset is always 0 while write_result.data_offset
-    // carries proper data offset in a blob.
-    // There should be an issue on the matter.
-#define CMP(x) BOOST_REQUIRE_EQUAL(std::get<0>(write_result).x, std::get<0>(lookup_result).x)
-    CMP(record_flags);
-    CMP(user_flags);
-    CMP(json_timestamp);
-    CMP(json_offset);
-    CMP(json_size);
-    CMP(json_capacity);
-    CMP(data_timestamp);
-    // CMP(data_offset);
-    CMP(data_size);
-#undef CMP
-
```

Comment:
This comparison is important, it checks that read_result bears the same record_info as write_ and lookup_ results (apart from data_offset member inequality.)
We decided that data_offset inequality is a wanted behavior, so FIXME-comment should have gone but check itself should have stayed.